README.md

```diff
@@ -362,12 +367,109 @@ The `$SERVICE_BINDING_ROOT` environment variable **MUST NOT** be reset if it is
 
 If a `.spec.type` is set, the `type` entry in the binding `Secret` **MUST** be set to its value overriding any existing value.  If a `.spec.provider` is set, the `provider` entry in the binding `Secret` **MUST** be set to its value overriding any existing value.
 
+### Binding Status
+
 If the modification of the Application resource is completed successfully, the `Ready` condition status **MUST** be set to `True`.  If the modification of the Application resource is not completed successfully the `Ready` condition status **MUST NOT** be set to `True`.
 
 # Extensions
 
 Extensions are optional additions to the core specification as defined above.  Implementation and support of these specifications are not required in order for a platform to be considered compliant.  However, if the features addressed by these specifications are supported a platform **MUST** be in compliance with the specification that governs that feature.
 
+## Custom Projection
+
+There are scenarios where the Reconciler that processes the bindings of a Provisioned Service (hereinafter referred to as `Reconciler A`) is different than the Reconciler that will project those bindings into the Application (hereinafter referred to as `Reconciler B`). To coordinate the projection between Reconcilers A and B the `ServiceBinding` CR author **MUST** set the `projection.service.binding` annotation to `Custom`.  
+
+Reconciler A reacts to this annotation by creating a new `ServiceBindingProjection` CR which includes the necessary information for Reconciler B, who is watching for and takes ownership of new `ServiceBindingProjection` resources (see the [Custom Projection Definition](#custom-projection-definition) section below), to carry out the projection.  Reconciler A is responsible for updating `ServiceBindingProjection` upon corresponding changes to `ServiceBinding`.
```

Comment:
my thought is that the different reconcilers would simply key off the apiVersion / Kind, rather than the annotation.  For example, the Runtime Component Operator would process any `ServiceBindingProjection` that had `application.kind` set to `RuntimeComponent`.  This would avoid the need for `ServiceBinding` authors to know all of the enums that could go into the annotation and when to use each one.

Reply:
Good discussion - it's important to explore the different use cases.  I can definitely see the value of that extra metadata for duck type. But I still go back to balancing user experience of `ServiceBinding` authors vs extra work for Reconcilers - always putting more weight to the former.

As a `ServiceBinding` author is probably easier for me to just add `projection.service.binding/type: Custom` (thanks for the suggestion!), than it is for me to know what metadata will trigger the correct Reconciler.  

From the Reconciler's (B) perspective, it should be able to check for duck type patterns just like others (like `status.binding.name`).  And if Reconciler B really wants to use metadata, maybe it relies on the authors to set the app via a labelSelector, and check for `ServiceBindingProjection`s that use a certain selector?

Reply:
I'm ok with this, we can loosen/expand if/when there is demand